client/src/components/BookingForm/index.js

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, Row, Typography, Result, Alert } from 'antd';
+import axios from 'axios';
+import './style.css';
+
+const { Title } = Typography;
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const BookingForm = () => {
+  const [success, setSuccess] = useState(false);
+  const [error, setError] = useState();
+
+  const onFinish = async ({
+    firstName,
+    lastName,
+    email,
+    birthday,
+    phone,
+    diseases,
+    appointmentDate,
+    appointmentTime,
+    complaints,
+  }) => {
+    try {
+      setSuccess(false);
+      await axios.post('/api/v1/appointments', {
+        firstName,
+        lastName,
+        email,
+        birthday,
+        phone,
+        diseases,
+        appointmentDate,
+        appointmentTime,
+        complaints,
+      });
+      setSuccess(true);
+    } catch (err) {
+      let e;
+      if (err.response.data.error === 'Unavailable Time') {
+        e = 'please choose another appointment time';
+      } else if (err.response.data.error === 'Validation Error') {
+        e = err.response.data.message;
+      } else if (err.response.data.error === 'RangeError') {
+        e = 'Input valid date in form YYYY-MM-DD or time hh:mm:ss ';
+      } else {
+        e = 'Sever error, please try again later';
+      }
+      setError(e);
+    }
+  };
+
+  return (
+    <div className="booking">
+      <Form {...layout} onFinish={onFinish} name="basic">
+        <Title color="#fff"> Book an appointment</Title>
+        {error && (
+          <Alert
+            className="err"
+            id="alert"
+            message={error}
+            type="error"
+            showIcon
+          />
+        )}
+        <Row>
+          <Form.Item
+            name="firstName"
+            rules={[{ required: true, message: 'Please input first Name!' }]}
+          >
+            <Input className="first-name" placeholder="First Name" />
+          </Form.Item>
+          <Form.Item
+            name="lastName"
+            rules={[{ required: true, message: 'Please input last name!' }]}
+          >
+            <Input className="last-name" placeholder="Last Name" />
+          </Form.Item>
+        </Row>
+        <Form.Item name="email">
+          <Input placeholder="Email" />
+        </Form.Item>
+        <Form.Item name="birthday">
+          <Input placeholder="Birthday" />
+        </Form.Item>
+        <Form.Item
+          name="phone"
+          rules={[
+            { required: true, message: 'Please input your phone number!' },
+          ]}
+        >
+          <Input placeholder="Phone" />
```

Comment:
The Phone is the main input in our application, as you know it should be unique and with a specific format, So I think you need to add some validation to it, so the user have to enter it in the same shape every time
+970 - 59 - 759-5893
By Apply this format, the user will enter the phone number in the correct format every time